tests/build/jvm-build.go

```diff
@@ -31,7 +32,7 @@ var (
 	testProjectRevision = utils.GetEnv("JVM_BUILD_SERVICE_TEST_REPO_REVISION", "main")
 )
 
-var _ = framework.JVMBuildSuiteDescribe("JVM Build Service E2E tests", Label("jvm-build", "HACBS"), func() {
+var _ = framework.JVMBuildSuiteDescribe("JVM Build Service E2E tests", Pending, Label("jvm-build", "HACBS"), func() {
```

Comment:
why is a test about the release-service turning off the jvm build test suite?

Reply:
was reverted